src/v/model/BUILD

```diff
@@ -13,6 +13,22 @@ redpanda_cc_library(
     ],
 )
 
+redpanda_cc_library(
```

Comment:
personally I always thought this enum should go in //src/v/compression and we can alias it in models if we want.

We currently do the otherway around, and I feel like it's backwards. Compression is a lower level primitive than our application models

Reply:
its a big change ill push another commit for it